linters/ruff/plugin.yaml

```diff
@@ -10,6 +10,7 @@ lint:
   definitions:
     - name: ruff
       files: [python, python-interface]
+      description: A Python linter and formatter
```

Comment:
Not sure if we want to call out formatter here, especially since it's not enabled by default